python/huggingfaceserver/huggingfaceserver/__main__.py

```diff
@@ -43,7 +55,7 @@ def list_of_strings(arg):
     help="Huggingface tokenizer revision",
 )
 parser.add_argument(
-    "--max_length", type=int, default=None, help="max sequence length for the tokenizer"
+    "--max_length", type=int, default=16, help="max sequence length for the tokenizer"
```

Comment:
Any reason to set the default value to 16? HF API is defaulting to None
https://huggingface.co/transformers/v2.11.0/main_classes/tokenizer.html#pretrainedtokenizer

Reply:
I'm not sure why I made it 16. I've changed it to use HF's default.